src/App.css

```diff
@@ -31,138 +34,240 @@
   width: 100%;
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
+  justify-content: flex-start;
   align-items: center;
+  background-color: #fff;
+  border-bottom: 2px solid #adaaaa;
+  padding-left: 10px;
 }
 
 .navbar h1 {
   color: #0290ff;
 }
 
-.navbar-menu {
+.user__profile__icon {
+  color: #0290ff;
+  border: 1px solid #888;
+  padding: 5px;
+  border-radius: 100%;
+  display: flex;
+  margin-left: 300px;
+}
+
+.nav__menu {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
```

Comment:
- [ ] Great effort working on the styling of the bookstore and some of the changes requested by the previous reviewer. But according to the previous reviewer, the user icon should be on the right-hand side and align with the books on a straight line vertically
- [ ] Also the `Add book` button in the add book form should be aligned on a straight line with the books added vertically. With this, you will be sure you replicate the design as seen on Zeplin perfectly.


| Your Design 👇| Zeplin Design 👇|
| ------------- | ------------- |
| ![book-emma](https://github.com/musangamfure/bookstore/assets/78634841/c7fe9f50-1215-448d-b33a-ba35ceea38e3)  | ![zep-org](https://github.com/musangamfure/bookstore/assets/78634841/5ce83a90-7fab-4e28-b9fc-b21e74376ff6)  |